requirements.txt

```diff
@@ -40,3 +40,4 @@ selenium==2.44.0
 urllib3==1.10
 wsgiref==0.1.2
 Werkzeug==0.11.3
+MySQL-python==1.2.5
```

Comment:
This library is small, 110k. It needs the mysql libs to install, so it won't work on os x or windows, but we have separate client building requirements.txt for those. So I think this is OK to go in.